docs/yaml/functions/response_file.yaml

```diff
@@ -0,0 +1,71 @@
+name: response_file
+returns: list[str] | str
+description: |
```

Comment:
Would be great to reference the ninja docs here as a "similar to rsp_file in Ninja" type thing.

Reply:
I referenced msvc instead. I think this is a better reference for that.